.gitignore

```diff
@@ -92,4 +92,10 @@ typings/
 # DynamoDB Local files
 .dynamodb/
 
+# Ide files
+.idea
+
+# Lock files
+/package-lock.json
```

Comment:
Nice but why add package-lock.json to the .gitignore file?

Reply:
Because it autogenerates upon dependencies install along with with `node_modules` folder, so no need to push it.

Reply:
According to npm it should be checked into source control. https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

Reply:
Let me remove it from .gitignore I saw, it was not meant to be ignored. I was a mistake.